pkg/ovalutil/rpm.go

```diff
@@ -105,3 +108,16 @@ func walkCriterion(ctx context.Context, node *oval.Criteria, cris *[]*oval.Crite
 		*cris = append(*cris, &c)
 	}
 }
+
+func getEnabledModules(cris []*oval.Criterion) []string {
+	enabledModules := []string{}
+	for _, criterion := range cris {
+		var regexComment = regexp.MustCompile(`(Module )(.*)( is enabled)`)
```

Comment:
I'd move this to an init function in the module, no need to build the regex var everytime this method is called. 
something like... 
```
package ovalutil

var regexComment

func init() {
  regexComment = regex.MustCompile(...)
}
```

Reply:
Fixed